### ISSUE TITLE
Panic when initializing env_logger when this provider is statically linked into a host

### ISSUE THREAD

**Issue by ewbankkit** [wascc/logging-provider#4]
If this capability provider is statically linked into a host which has itself initialized logging there is a panic at

https://github.com/wascc/logging-provider/blob/0edbc4903426f309001bd2812fbdefad79d26573/src/lib.rs#L45-L47

```

2020-04-03T15:10:17.267-04:00 | note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
2020-04-03T15:10:17.745-04:00 | START RequestId: 12873151-6a75-42f8-9cfd-3a0c0e06c902 Version: $LATEST
2020-04-03T15:10:17.748-04:00 | [INFO bootstrap] aws-lambda-wascc-runtime starting
2020-04-03T15:10:17.748-04:00 | [INFO provider] Logger already intialized
2020-04-03T15:10:17.748-04:00 | thread 'main' panicked at 'env_logger::init should not be called after logger initialized: SetLoggerError(())', src/libcore/r
```

**Comment by ewbankkit** [comment]
Suggest replacing with something like

```rust
        if env_logger::try_init().is_err() {
            info!("Logger already intialized");
        }
```

**Comment by autodidaddict** [comment]
Fixed in upcoming 0.6 builds...
